### ISSUE TITLE
Counting MAUs

### ISSUE THREAD

**Issue by incloudss** [heremaps/here-sdk-examples#250]
Hi,

I've developed the flutter app (it is released on Google Play already) that uses HERE Navigate Edition. 
I am using currently old, 4.7.6 version. The business model in my app assumes, that only premium users, paying 5$/month, will be able to get HERE nagivation. Free users do not have HERE services (including navigation and map) at all. 
To avoid counting free users as MAUs, i am not initializing the SDKContext and SDKNativeEngine. The HERE credentails are not even accessible for them (creds are fetched from server, and free user is not authorized to do so).

From what i see from usage, it works as expected. But here is one thing. I am reading newer documentation for version 4.14.0 of sdk, and it says: 
**"Note: The time of initialization of the HERE SDK does not have an impact on how transactions and monthly active users (MAU) are counted."** 
Does it mean that not initializing HERE SDK won't prevent MAU counting or am i wrong? How is that possible, if the user does not even have access to HERE credentials?

**Comment by HERE-SDK-Support-Team** [comment]
Hi @incloudss, yes, this is correct: We changed the way how MAU are counted: https://developer.here.com/documentation/flutter-sdk-navigate/dev_guide/topics/billing.html

Initialization of the HERE SDK does no longer count. We count only per feature: "A feature is executed, when a MapView is shown, or when any engine like SearchEngine, RoutingEngine or any other engine - including MapDownloader, RoutePrefetcher, Navigator and VisualNavigator (for users of the Navigate Edition) - is instantiated. Resuming an application does not count as separate start. The instantiation of the HERE SDK itself is also not counted."

Basically, any of these features requires that the HERE SDK is initialized. If the HERE SDK is not initialized, you cannot use, for example, the `SearchEngine`: The search feature will only work, when you have set valid credentials when initializing the HERE SDK - and these are used subsequently by any feature or engine.

For your case, when using HERE SDK >= 4.12.0.0, it indeed does not matter if you init the HERE SDK for the Navigate Edition or not. Instead, when using the `MapDownloader` or using the `Navigator` it will be counted as MAU.

Hope this clarifies. Leave a reply if you have more questions. Thank you!
